examples/showoff.sil

```diff
@@ -4,6 +4,7 @@
 \script[src=packages/image]
 \script[src=packages/bidi]
 \set[parameter=linebreak.tolerance,value=10000]
+\noindent
```

Comment:
@simoncozens I expected this to fix the title not being centered ... and indeed it did move it closer to center (see #625). However to my surprise it did not _entirely_ fix the issue. The title is s till not quite centered and I'm not sure where the extra leading space is coming from now. Any ideas?

Reply:
This was because of the leading whitespace on lines like this:
```
\begin{pagetemplate}
    \frame{}
    \frame{}
\end{pagetemplate}
```
The spaces were putting stuff in the node queue before starting to typeset the centered content.

I've mitigated this two ways:

1. Discard all content inside pagetemplates. The `\frame` commands still work, not much else.
2. Throw warnings if `\center` or `\nodindent` are used after things are in the horizontal node queue.